src/sev-snp/sev_support.cc

```diff
@@ -46,6 +46,7 @@
 using namespace certifier::framework;
 using namespace certifier::utilities;
 
+// Had to sudo chmod 644 this to allow non-root to open this device
```

Comment:
Let's remove this comment. It's not necessary.

Reply:
Done. (Oversight ...)